pyproject.toml

```diff
@@ -48,3 +51,19 @@ disable = [
 [build-system]
 requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.mypy]
+exclude = ".venv"
+strict = true
+
+[[tool.mypy.overrides]]
+module = [
+    "apy.anki",
+    "apy.cli",
+    # "apy.config",
```

Comment:
Perhaps we could add a comment above line 59 to explain wht the point of these overrides are? And I guess we can _delete_ the finished lines instead of commenting them out?